entrypoint.sh

```diff
@@ -9,8 +9,15 @@ defaultContext = "default"
   [Contexts.default]
     clientId = "${INPUT_CLIENT_ID}"
     clientSecret = "${INPUT_CLIENT_SECRET}"
-    accessToken = "${INPUT_ACCESS_TOKEN}"
-    project = "${INPUT_PROJECT}"
 EOF
 
-sloctl apply -f "${INPUT_SLOCTL_YML}"
+flags=(
+  -f "${INPUT_SLOCTL_YML}"
+  -y # default to auto confirm
```

Comment:
Let's maybe leave some context here:

```suggestion
  -y # Required to auto confirm, for more details refer to: https://docs.nobl9.com/sloctl-user-guide?_highlight=prompt&_highlight=threshold#apply
```